packages/accordion/accordion.scss

```diff
@@ -13,16 +13,25 @@
 }
 
 @include jkl.comfortable-density-variables {
-    --jkl-accordion-title-padding: #{jkl.$spacing-m};
-    --jkl-accordion-title-text-margin-left: #{jkl.rem(18px)};
-    --jkl-accordion-content-padding: #{jkl.$spacing-m};
+    --jkl-accordion-title-padding: #{jkl.$spacing-m} #{jkl.rem(48px)} #{jkl.$spacing-m} #{jkl.$spacing-m};
```

Comment:
> `jkl.rem(48px)`

Går det an å omformulere denne som pilens bredde pluss en Jøkul-spacing, eller tilvarende? Så blir man kvitt et magisk tall..

Reply:
Er ikke noe som går fint opp i en verdi fra spacingskalaen dessverre (foo + spacing-n). Så da blir det foo, bar og baz + spacing-a og spacing-b (for compact). Ser ikke helt verdien.